test/tests/parser/parse_int64_t/parse_int64_t.i

```diff
@@ -0,0 +1,16 @@
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 3
+    ny = 3
+    xmin = 0
+    xmax = 3
+    ymin = 0
+    ymax = 3
+    extra_element_integers = test_id
+    extra_element_integer_ids = '0 -1 2
+                                 1  3 450359962
+                                -2  3 18446744073709551614'
```

Comment:
Could you make this test use the vector of vectors? E.g. add multiple `extra_element_integers`

Reply:
Yes.